Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -502,6 +502,18 @@ theorem norm_le_mul_norm_add (u v : E) : ‖u‖ ≤ ‖u * v‖ + ‖v‖ :=
     ‖u‖ = ‖u * v / v‖ := by rw [mul_div_cancel_right]
     _ ≤ ‖u * v‖ + ‖v‖ := norm_div_le _ _
 
+@[to_additive]
+lemma norm_mul_eq_norm (x : E) {y : E} (h : ‖y‖ = 0) : ‖x * y‖ = ‖x‖ := by
+  apply le_antisymm ?_ ?_
+  · simpa [h] using norm_mul_le' x y
+  · simpa [h] using norm_le_mul_norm_add x y
+
+@[to_additive]
+lemma norm_div_eq_norm (x : E) {y : E} (h : ‖y‖ = 0) : ‖x / y‖ = ‖x‖ := by
+  apply le_antisymm ?_ ?_
+  · simpa [h] using norm_div_le x y
+  · simpa [h] using norm_sub_norm_le' x y
```

Comment:
Can you add `left` and `right` versions here, as you did with `inner`?

Reply:
for `norm_mul_eq_norm` I did it. what is the `left` version of `div`? or do you mean `‖x⁻¹ * y‖ = ‖y‖`?

Reply:
I think your `div` version is fine. You can add the `inv` versions if you want; it's up to you.

Reply:
I meant `(h : ‖x‖ = 0) : ‖x / y‖ = ‖y‖`